src/definitions/behaviors/form.js

```diff
@@ -8,2070 +8,2029 @@
+
+                                                            break;
+
+                                                        case 'datetime':
+                                                            values[name] = settings.formatter.datetime(date);
+
+                                                            break;
+
+                                                        case 'time':
+                                                            values[name] = settings.formatter.time(date);
+
+                                                            break;
+
+                                                        case 'month':
+                                                            values[name] = settings.formatter.month(date);
+
+                                                            break;
+
+                                                        case 'year':
+                                                            values[name] = settings.formatter.year(date);
+
+                                                            break;
+
+                                                        default:
+                                                            module.debug('Wrong calendar mode', $calendar, type);
+                                                            values[name] = '';
+                                                    }
+                                                }
+                                            } else {
+                                                values[name] = '';
+                                            }
+                                        } else {
+                                            values[name] = value;
+                                        }
+                                    }
+                                }
+                            });
+
+                            return values;
+                        },
+                        dirtyFields: function () {
+                            return $field.filter(function (index, e) {
+                                return $(e).data(metadata.isDirty);
+                            });
+                        },
+                    },
+
+                    has: {
+
+                        field: function (identifier) {
+                            module.verbose('Checking for existence of a field with identifier', identifier);
+                            identifier = module.escape.string(identifier);
+                            if (typeof identifier !== 'string') {
+                                module.error(error.identifier, identifier);
+                            }
+
+                            return (
+                                $field.filter('#' + identifier).length > 0 ||
+              $field.filter('[name="' + identifier + '"]').length > 0 ||
+              $field.filter('[data-' + metadata.validate + '="' + identifier + '"]').length > 0
```

Comment:
indent is completely missing

Reply:
fixed - eslint does not indent lines starting/ending with an operator